linalg/densemat.hpp

```diff
@@ -523,6 +568,12 @@ void AddMult_a_VWt(const double a, const Vector &v, const Vector &w,
 /// VVt += a * v v^t
 void AddMult_a_VVt(const double a, const Vector &v, DenseMatrix &VVt);
 
+/// Computes matrix P^t * A * P
```

Comment:
We should add notes that `RAP` will be resized.

Reply:
done!